wiki/Tournaments/OVMT/1/en.md

```diff
@@ -0,0 +1,351 @@
+| **1x 100** ![][flag_CZ] | **1** | 0 | ![][flag_RO] MaddaFakka-sama | *forfeit* |
+| **NyarkoO** ![][flag_CZ] | **4** | 2 | ![][flag_GB] Dovahcake |  |
+| **YunoWhoItIs** ![][flag_GB] | **4** | 1 | ![][flag_US] MicMck101 |  |
+| **Bubbleman** ![][flag_GB] | **4** | 0 | ![][flag_GB] Lewd |  |
+
+Tuesday, 13 September 2016:
+
+| Player 1 | | | Player 2 |
+| --: | :-: | :-: | :-- |
+| Seraxis ![][flag_GB] | 2 | **4** | ![][flag_FI] **Kissatyty** |
+
+### Round of 64
+
+| Player 1 | | | Player 2 | Match info |
+| --: | :-: | :-: | :-- | :-- |
+| Rei yo ![][flag_DE] | 0 | **1** | ![][flag_GB] **Lewd** | *forfeit* |
+| Blegie ![][flag_DE] | 2 | **3** | ![][flag_CZ] **1x 100** |  |
+| Mirt ![][flag_AU] | 0 | **1** | ![][flag_RO] **MaddaFakka-sama** | *forfeit* |
+| dika312  ![][flag_ID] | 0 | **1** | ![][flag_GB] **YunoWhoItIs** | *forfeit* |
+| **MicMck101** ![][flag_US] | **1** | 0 | ![][flag_DE] xXMalvinXx | *forfeit* |
+| -Lolisim- ![][flag_US] | 0 | **3** | ![][flag_GB] **WufWuf** |  |
+| **jamimc2** ![][flag_AU] | **1** | 0 | ![][flag_HU] scuoa  | *forfeit* |
+| **Sam** ![][flag_GB] | **3** | 0 | ![][flag_CL] \[Burno-\] |  |
+| Zetopweaboo ![][flag_AU] | 0 | **3** | ![][flag_GB] **Jerome** |  |
+| Sexual Tentacle ![][flag_CA] | 0 | **3** | ![][flag_GB] **SEXaris** |  |
+| RealXantoo ![][flag_AT] | 0 | **3** | ![][flag_FI] **Kissatyty** |  |
+| **NyarkoO** ![][flag_CZ] | **3** | 0 | ![][flag_CA] Jakesoly |  |
+| icecream-chan ![][flag_GB] | 1 | **3** | ![][flag_GB] **Dovahcake** |  |
+| Synxx3 ![][flag_DE] | 0 | **3** | ![][flag_US] **Catgirl** |  |
+| Han106 ![][flag_US] | 0 | **3** | ![][flag_RU] **u624ka** |  |
+| TropicalX ![][flag_GB] | 0 | **3** | ![][flag_FI] **Sanakirja** |  |
+| **BitComet** ![][flag_FI] | **3** | 2 | ![][flag_GB] mjozog3 |  |
+| **at-cay** ![][flag_US] | **3** | 0 | ![][flag_GB] Dawnsday |  |
+| nahh ![][flag_GB] | 0 | **1** | ![][flag_US] **Kradfiz** | *forfeit* |
+| **Kosmit** ![][flag_PL] | **3** | 0 | ![][flag_SG] handsome |  |
+| **Lefafel** ![][flag_FI] | **3** | 2 | ![][flag_GB] Tattoos |  |
+| **Doughy** ![][flag_GB] | **3** | 1 | ![][flag_GB] NotToBlame |  |
+| M-O-P-S-I ![][flag_FI] | 1 | **3** | ![][flag_SE] **Nitroz** |  |
+| Hasert ![][flag_MY] | 0 | **3** | ![][flag_GB] **Doomsday is Bad** |  |
+| **Muuki** ![][flag_FI] | **3** | 2 | ![][flag_US] Squilly |  |
+| **Pelzio** ![][flag_GB] | **3** | 0 | ![][flag_AU] TheJimnebob |  |
+| **- Auto -** ![][flag_CA] | **3** | 0 | ![][flag_GB] Khunee |  |
+| **Babo-ya** ![][flag_FI] | **3** | 0 | ![][flag_ES] TriFerlol |  |
+| **Halfslashed** ![][flag_US] | **3** | 0 | ![][flag_PL] \[-KuleK-\] |  |
+| **dong** ![][flag_GB] | **3** | 1 | ![][flag_US] YouKittenMe |  |
+
+## Ruleset
+
+- This tournament is a 1v1 single-elimination international osu! tournament.
+- This tournament will consist of 64 players.
+- Your rank must be lower than #100,000 to play. Signups under this rank will not be considered.
```

Comment:
is the wording on these sentences correct? currently either ambiguous or confusing

```suggestion
- Your rank must be lower than #100,000 to play. Sign-ups under this rank will not be considered.
```

Reply:
It was the wording that was used at the time 🤷

Reply:
```suggestion
- Your rank must be better than #100,000 to play. Sign-ups worse this rank will not be considered.
```
This could work and is essentially in line with how OWC words their rank rule,

Reply:
seems ok